puppetboard/Dockerfile

```diff
@@ -19,13 +21,12 @@ RUN apk add --no-cache --update py-pip && \
 
 RUN pip install puppetboard=="$PUPPET_BOARD_VERSION" gunicorn=="$GUNICORN_VERSION"
 
-COPY wsgi.py /var/www/puppetboard/
-COPY settings.py /var/www/puppetboard/
+COPY wsgi.py settings.py /var/www/puppetboard/
 
-EXPOSE 8000
+EXPOSE $PUPPETBOARD_PORT
 
 WORKDIR /var/www/puppetboard
 
-CMD ["/usr/bin/gunicorn", "-b", "0.0.0.0:8000", "wsgi:application"]
+CMD /usr/bin/gunicorn -b 0.0.0.0:${PUPPETBOARD_PORT} wsgi:application
```

Comment:
This is not in the JSON array format because we need the shell to parse the environment variable ($PUPPETBOARD_PORT)